erpnext/controllers/taxes_and_totals.py

```diff
@@ -70,6 +70,7 @@ def calculate_item_values(self):
 				item.net_rate = item.rate
 				item.amount = flt(item.rate * item.qty,	item.precision("amount"))
 				item.net_amount = item.amount
+				item.total_weight = flt(item.weight_per_unit * item.qty)
```

Comment:
Total weight will calculate only if there is taxes? I think this has to be done on qty change trigger?